### PR TITLE
Issue 21 context boundary

### PR DESCRIPTION
**Pull request by Weltraumschaf**

### CODE REVIEW

**Comment by Weltraumschaf** [comment]
@malexmave I added some more stuff. I'm a little bit stuck on the building block view. IMHO we should do a short meeting next week to discuss the building blocks.

For the moment I think a "syntactical review" is sufficient. For the over all consistency we should do a architecture review w/ some core developer to discuss the chapters.

**Comment by malexmave** [comment]
grafik

Can you double-check the links and fix the broken ones? I am not sure what the new URL scheme looks like.

File: docs/architecture/07_deployment_view.md [hunk 1]
```diff
@@ -9,28 +9,40 @@ sidebar_position: 7
...
+
 ### Motivation
 
-The motivation behind this scenario is to have one central point to accumulate all findings of all scanned namespaces. Typically, this scenario is for a team which want to monitor a whole landscape of applications and services (e.g. a SOC team). 
+The motivation behind this scenario is to have one central point to accumulate all findings of all scanned namespaces. Typically, this scenario is for a team which wants to monitor a whole landscape of applications and services (e.g. a security operations (SOC) team). 
 
 ## Cluster/Namespace Internal {#_cluster_namespace_internal}
 
-Cluster internal security scans directly in the business service's namespace.
+Cluster internal security scans directly in the business service's namespace. Only the _engine_ (_operator_, _lurker_, _hooks_) are deployed in a central dedicated namespace. The _Scan Types_ are deployed into the namespaces of the scanned application.
```
Comment (malexmave) [inline docs/architecture/07_deployment_view.md:32]: > Only the _engine_ (_operator_, _lurker_, _hooks_) are deployed in a central dedicated namespace.

Is this correct? I thought hooks are also installed into the sub-namespaces, and only the operator (and potentially lurker) are installed globally?

**Review by malexmave — CHANGES_REQUESTED** [review]
@Weltraumschaf I have made some style and grammar fixes and pushed them to this branch, please check if you are fine with them.

Please ensure that the project builds by fixing the ADR links in the blog post (see the previous comment). Other than that, I only have one question, seen as a comment below.

File: docs/architecture/07_deployment_view.md (hunk 1 above)
Comment (Weltraumschaf) [reply docs/architecture/07_deployment_view.md:32]: Yes, I'm pretty sure, but this is something we should review with jannik.robert.

About the ADR links: Of course. I wonder why it built on my machine...

Comment (malexmave) [reply docs/architecture/07_deployment_view.md:32]: Even easier: I checked our own internal scan git, we install hooks in the individual namespaces.

**Review by malexmave — APPROVED** [review]

**Pull request merged**